inspector/transform.py

```diff
@@ -1,7 +1,10 @@
+import json
 import os
 import re
 import requests
 
+from geekbench import geekbench_html_to_json
```

Comment:
Oh, one more thing. This should be moved into `fetch_geekbench_results`, so we can on-demand import all of those stuff to keep memory usage low.